article/app/views/fragments/liveBlogNavigation.scala.html

```diff
@@ -22,7 +22,7 @@
             </a>
```

Comment:
as discussed this is always page 1 which should navigate to the top, as there's no nagivation available to snap down to